canonical-web-and-design/jaas-dashboard
# Add google analytics tracking

## Done

- Add Google Analytics tracking
- Add GA event for filter panel visibility
- Add GA event for user panel visibility
- Add GA event for user expanding topology

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Sanity check code
-  Remove the `if (process.env.NODE_ENV === "production")` conditional
- Open demo link and verify yourself in realtime in GA dashboard

## Details

<img width="987" alt="Screenshot 2020-02-12 at 12 32 27" src="https://user-images.githubusercontent.com/505570/74335321-d6f2f780-4d93-11ea-927c-6cb534d9a104.png">

Fixes #341
